<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ShowOrderPapers</class>
 <widget class="QDialog" name="ShowOrderPapers">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>691</width>
    <height>535</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Listar Comandas | Sistema de Controle de Comandas</string>
  </property>
  <widget class="QPushButton" name="pushButton_searchPeriodOrderPapers">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>150</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Pesquisar</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>80</y>
     <width>651</width>
     <height>59</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_showOrderPapersPeriod">
    <item>
     <widget class="QLabel" name="label__showOrderPapersPeriodPeriodTitle">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Selecionar período</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_showOrderPapersPeriod">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout__showOrderPapersPeriodFromDate">
        <item>
         <widget class="QLabel" name="label_fromDate">
          <property name="text">
           <string>De: </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDateTimeEdit" name="dateTimeEdit_fromDate"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_showOrderPapersPeriodToDate">
        <item>
         <widget class="QLabel" name="label_toDate">
          <property name="text">
           <string>Até: </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDateTimeEdit" name="dateTimeEdit_toDate"/>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label_showOrderPapersTitle">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>641</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Listar Comandas</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidget_showOrderPapers">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>200</y>
     <width>641</width>
     <height>311</height>
    </rect>
   </property>
   <property name="editTriggers">
    <set>QAbstractItemView::NoEditTriggers</set>
   </property>
   <property name="showGrid">
    <bool>true</bool>
   </property>
   <attribute name="horizontalHeaderVisible">
    <bool>true</bool>
   </attribute>
   <attribute name="horizontalHeaderStretchLastSection">
    <bool>true</bool>
   </attribute>
   <attribute name="verticalHeaderVisible">
    <bool>false</bool>
   </attribute>
   <column>
    <property name="text">
     <string>ID</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Cliente</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Entrada</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Pedidos</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Valor Total</string>
    </property>
   </column>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
